Source/WebCore/dom/PointerEvent.cpp

```diff
@@ -293,4 +296,39 @@ void PointerEvent::receivedTarget()
         predictedEvent->setTarget(this->target());
 }
 
+bool PointerEvent::fractionalCoordinatesAllowedForType(const AtomString& type)
```

Comment:
This doesn't have to be a class method

Reply:
i.e.

```cpp
static bool fractionalCoordinatesAllowedForType(const AtomString& type)
{
    ....
}
```

Reply:
oh, my bad, I missed its use there.

Reply:
Although, given this does not depend on a `PointerEvent` instance state, maybe make it a static class method. Totally nit though